shub/image/utils.py

```diff
@@ -31,10 +31,13 @@
 """
 
 
-def debug_log(msg):
+def is_verbose():
     ctx = click.get_current_context(True)
-    # FIXME remove check of debug param as it's deprecated
-    if ctx and (ctx.params.get('verbose') or ctx.params.get('debug')):
+    return ctx and ctx.params.get('verbose')
```

Comment:
We can't remove the check for `debug` unless we also remove the `--debug` parameter from all the `shub image` commands (which in turn we can't do unless we release v3.0 because we follow semantic versioning).

We should either leave the `debug` check here or alternatively we can remove it and add a callback for the `--debug` parameters that sets `ctx['verbose'] = True`